pkg/api/query/v1.go

```diff
@@ -701,22 +713,29 @@ func (qapi *QueryAPI) query(r *http.Request) (interface{}, []error, *api.ApiErro
 		return nil, nil, &api.ApiError{Typ: api.ErrorExec, Err: res.Err}, qry.Close
 	}
 
+	if seriesStats != nil {
+		qapi.logger.Log("series stats", fmt.Sprintf("%+v", seriesStats))
```

Comment:
nit: use `level.Debug(logger).Log` API to control which log it is, this is debug log IIUC and could be very verbose in production